_data/conferences.json

```diff
@@ -0,0 +1,22 @@
+[
+  {
+    "conference_name": "PyconUg",
+    "url": "https://ug.pycon.org",
+    "conference_start_date": "2024-10-09",
+    "conference_end_date": "2024-10-13",
+    "conference_type": "in-person",
+    "conference_location": "NWSC International Resource Centre (IREC), Kampala, Uganda",
+    "summary": "The conference is a five-day event, 9-13 October. PyconUganda inherit this tradition of an annual event that brings together python enthusiasts, tech audiences and technology business in uganda and around the world.",
+    "speaking": null
```

Comment:
I'm curious, what will the format of the `speaking` field look like? Would it be a list of json objects?

Reply:
Currently it is a  string of entries...

I was thinking of a GH-Issue feature that would that would convert a list to a python list.

with events.md being a .md file. It should render properly in the website.